### PR TITLE
docs: Node.js add RESTful docs

### PR DESCRIPTION
**Pull request by xleili**

### CODE REVIEW

File: docs-cn/14-reference/03-connector/node.mdx [hunk 1]
```diff
@@ -201,3 +201,53 @@ promise2.then(function (result) {
 [node-example-raw.js](https://github.com/taosdata/tests/tree/master/examples/nodejs/node-example-raw.js)同样是一个使用 NodeJS 连接器建表，插入天气数据并查询插入的数据的代码示例，但和上面不同的是，该示例只使用`cursor`。
 
 [所有示例代码](https://github.com/taosdata/TDengine/tree/develop/src/connector/nodejs/examples)
+
+## 使用RESTful 连接器
```
Comment (sangshuduo) [inline docs-cn/14-reference/03-connector/node.mdx]: need a space between Chinese and English

File: docs-cn/14-reference/03-connector/node.mdx [hunk 2]
```diff
@@ -201,3 +201,53 @@ promise2.then(function (result) {
...
+TDengine 还提供TypeScript的RESTful 连接器。
```
Comment (sangshuduo) [inline docs-cn/14-reference/03-connector/node.mdx]: space

**Review by sangshuduo — CHANGES_REQUESTED** [review]
see my comments

File: docs-cn/14-reference/03-connector/node.mdx (hunk 1 above)
Comment (dingbo8128) [reply docs-cn/14-reference/03-connector/node.mdx:205]: I will fix it in another pr

**Review by dingbo8128 — APPROVED** [review]
LGTM

File: docs-cn/14-reference/03-connector/node.mdx [hunk 3]
```diff
@@ -228,26 +234,29 @@ let cursor = conn.cursor();
...
     result.getErrCode()
-    // 获得执行的命令的错误信息，返回string|undefined（当执行SQL语句失败时返回undefined）。
+    // 获得执行的命令的错误信息，返回 string|undefined（当执行 SQL 语句失败时返回 undefined）。
     result.getErrStr()
 ```
+
 ### 示例程序
-更多示例参考[TDengineREST connector](https://github.com/taosdata/TDengine/tree/develop/src/connector/TypeScript-REST)
+
+更多示例参考 [TDengineREST connector](https://github.com/taosdata/TDengine/tree/develop/src/connector/TypeScript-REST)。
```
Comment (sangshuduo) [inline docs-cn/14-reference/03-connector/node.mdx]: TypeScript REST connector?

**Review by sangshuduo — COMMENTED** [review]
commented

**Review by sangshuduo — APPROVED** [review]
LGTM

**Pull request merged**